src/main/java/com/facebook/openwifirrm/modules/ProvMonitor.java

```diff
@@ -105,29 +110,58 @@ private void runImpl() {
 		// ignoring "entity" completely
 		InventoryTagList inventory = client.getProvInventory();
 		SerialNumberList inventoryForRRM = client.getProvInventoryForRRM();
-		VenueList venueList = client.getProvVenues();
 		//EntityList entityList = client.getProvEntities();
 		if (inventory == null || inventoryForRRM == null) {
 			logger.error("Failed to fetch inventory from owprov");
 			return;
 		}
+
+		// fetch the RRM details for each venue
+		Map<String, RRMDetails> rrmDetails = new HashMap<String, RRMDetails>();
```

Comment:
nit: you can omit the `<>` generic arguments in the instantiation when it can be inferred:
```java
Map<String, RRMDetails> rrmDetails = new HashMap<>();
```